src/rules/util.js

```diff
@@ -99,6 +99,13 @@ const testCaseNames = Object.assign(Object.create(null), {
   xtest: true,
 });
 
+const testHookNames = Object.assign(Object.create(null), {
```

Comment:
Can we use a Set for this? (and change the code above as well)
This way of doing it is probably from before Sets were supported everywhere.

Reply:
I thought it was the standard followed in the project, will change it

Reply:
It probably was so far, but I imagine only for historical reasons so it's a good time to change that when touching this code :)